spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -38,6 +57,106 @@
           expect(cur_plat._platform.target_user).to eq(plat[:target_user])
         end
       end
+
+      describe '#generate_msi_packaging_artifacts' do
+        before do
+          # Create Workdir and temp root directory
+          FileUtils.mkdir_p("#{workdir}/wix")
+          FileUtils.mkdir_p("#{VANAGON_ROOT}/resources/windows/wix")
+          FileUtils.mkdir_p("#{PROJ_ROOT}/resources/windows/wix")
+          # Need to set proj_root to our fixtures/sample directory
+          cur_plat._platform.proj_root = PROJ_ROOT
+        end
+
+        it "copies Wix File from product specific directory to output directory" do
+          # setup source directories and run artifact generation
+          FileUtils.cp("#{wixtestfiles}/file-1.wxs", "#{PROJ_ROOT}/resources/windows/wix/file-1.wxs")
+          cur_plat._platform.generate_msi_packaging_artifacts(workdir,plat[:projname], binding)
+          # check the result
+          expect(File).to exist("#{workdir}/wix/file-1.wxs")
+        end
+
+        it "Copies file Wix File from Vanagon directory to work directory" do
+          # setup source directories and run artifact generation
+          FileUtils.cp("#{wixtestfiles}/file-1.wxs", "#{VANAGON_ROOT}/resources/windows/wix/file-1.wxs")
+          cur_plat._platform.generate_msi_packaging_artifacts(workdir,plat[:projname], binding)
+          # check the result
+          expect(File).to exist("#{workdir}/wix/file-1.wxs")
+        end
+
+        it "Pick Project Specific File in favour of Generic file" do
```

Comment:
as just discussed, these titles could use a bit copy-editing: all should be lower cased and third-person active.